spec/views/post_show_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'rails_helper'
+
+RSpec.describe 'posts/show', type: :feature do
+  before(:each) do
+    @user1 = User.create!(
+      name: 'username1',
+      photo: 'Photo',
+      bio: 'Text',
+      posts_counter: 2
+    )
+    @user2 = User.create!(
+      name: 'username2',
+      photo: 'Photo2',
+      bio: 'Text2',
+      posts_counter: 0
+    )
+    @post1 = Post.create!(
+      title: 'Title1',
+      text: 'Text1',
+      comments_counter: 3,
+      likes_counter: 1,
+      author: @user1
+    )
+    @post2 = Post.create!(
+      title: 'Title2',
+      text: 'Text2',
+      comments_counter: 0,
+      likes_counter: 0,
+      author: @user1
+    )
+    @comment1 = Comment.create!(
+      text: 'Comment1',
+      post: @post1,
+      author: @user2
+    )
+    @comment2 = Comment.create!(
+      text: 'Comment2',
+      post: @post1,
+      author: @user2
+    )
+    @comment3 = Comment.create!(
+      text: 'Comment3',
+      post: @post1,
+      author: @user2
+    )
+
+    visit user_post_path(@user1.id, @post1.id)
+  end
+
+  it 'Show the post title' do
+    expect(page).to have_content('Title1')
+  end
+
+  it 'Show the post author' do
+    expect(page).to have_content('username1')
+  end
+
+  it 'Show the number of comments' do
+    expect(page).to have_content('3')
+  end
+
+  it 'Show the number of likes' do
+    expect(page).to have_content('1')
+  end
+
+  it 'Show the post body' do
+    expect(page).to have_content('MyText1')
+  end
```

Comment:
- [ ] Kindly make this test pass. It's failing at the moment

![post-body](https://user-images.githubusercontent.com/26597432/233071253-e7ef5219-e647-4ec0-86fb-d0147577ecd6.png)